data/transactions/logic/assembler.go

```diff
@@ -1986,7 +1977,7 @@ func (ops *OpStream) assemble(text string) error {
 	if ops.Errors != nil {
 		l := len(ops.Errors)
 		if l == 1 {
-			return errors.New("1 error")
+			return fmt.Errorf("1 error: %s", ops.Errors[0].Error())
```

Comment:
I like this, will make for easier unit test debugging, but worry it'll change output somewhere. How about putting in a "\n" after the "1 error:"?
Also, consider using "%w" instead of "%s" (and not calling "Error()")